react_training/src/Assignment5/Question7.jsx

```diff
@@ -0,0 +1,57 @@
+// Create a form in your React component that allows the user to input data (e.g., a post title and content). Use Axios to send a POST request to an API endpoint, posting the user's data.
+
+import React, { useState } from "react";
+import axios from "axios";
+
+const Question7 = () => {
+  const initialData = { title: "", content: "" };
```

Comment:
Just something to note here
if `initialData` variable is not being used anywhere else this could have been written as 
`const [data, setData] = useState({ title: "", content: "" });`
just more cleaner